src/StudyApi/Common/RequestModelValidatorFilter.cs

```diff
@@ -28,7 +28,7 @@ public async Task OnActionExecutionAsync(ActionExecutingContext context, ActionE
                 _logger.LogWarning("Request Validation Failed: {Errors}", errorsString);
 
                 // Set a generic error message if not in development
-                if (!_environment.IsDevelopment())
+                if (!_environment.IsEnvironment("local") || !_environment.IsDevelopment())
```

Comment:
We should avoid sprinkling these environment checks throughout the code and setup the environment with dependency injection.